example/logs/kubernetes/single-tenant-no-masking.river

```diff
@@ -0,0 +1,108 @@
+/*
+The following items would need to be defined to include your own specific steps,
+this example removes the following modules:
+
+  - masking
+  - normalize filename
+
+*/
+logging {
+  level  = "info"
+  format = "logfmt"
+}
+
+module.git "log_targets" {
+  repository = "https://github.com/grafana/agent-modules.git"
+  revision = "main"
+  path = "modules/kubernetes/logs/targets/logs-from-worker.river"
+
+  arguments {
+    forward_to = module.git.log_formats_all.exports.process.receiver
+    tenant = "<YOUR_TENANT_NAME>"
```

Comment:
Not sure how I feel about placeholders here though it is an example. IMO if we use a placeholder it should fail to run so its very explicit. @erikbaranowski thoughts?

Reply:
for an example, I'd replace it with an environment var instead so for example something like:

`env("TENANT_NAME")`

Reply:
Updated examples